javascript/javascript2/week1/hyfBayHelpers.js

```diff
@@ -0,0 +1,72 @@
+/* DONT MODIFY ANY OF THIS CODE!!!*/
+
+window.getAvailableProducts = function() {
+  function getRandomInt(min, max) {
+    min = Math.ceil(min);
```

Comment:
Consider defining local variables instead of using min and max. In general, it is better not to change or edit function parameters to avoid any surprises.